src/modules/webhooks/get-interaction/1.0/module.json

```diff
@@ -1,40 +1,1053 @@
+      "defaultValue": "sourceDetailsPhoneNumberFormattedType"
+    },
+    {
+      "name": "Source Details Phone Number Formatted Number Type",
+      "value": "$.interaction.sourceDetails.phoneNumberFormatted.numberType",
+      "defaultValue": "sourceDetailsPhoneNumberFormattedNumberType"
+    },
+    {
+      "name": "Source Details Phone Number Formatted Country Calling Code",
+      "value": "$.interaction.sourceDetails.phoneNumberFormatted.countryCallingCode",
+      "defaultValue": "sourceDetailsPhoneNumberFormattedCountryCallingCode"
+    },
+    {
+      "name": "Source Details Phone Number Formatted National Number",
+      "value": "$.interaction.sourceDetails.phoneNumberFormatted.nationalNumber",
+      "defaultValue": "sourceDetailsPhoneNumberFormattedNationalNumber"
+    },
+    {
+      "name": "Source Details Phone Number Formatted Number",
+      "value": "$.interaction.sourceDetails.phoneNumberFormatted.number",
+      "defaultValue": "sourceDetailsPhoneNumberFormattedNumber"
+    },
+    {
+      "name": "Source Details Phone Number Formatted Country",
+      "value": "$.interaction.sourceDetails.phoneNumberFormatted.country",
+      "defaultValue": "sourceDetailsPhoneNumberFormattedCountry"
+    },
+    {
+      "name": "Source Details Phone Number Formatted Number Parts Area Code",
+      "value": "$.interaction.sourceDetails.phoneNumberFormatted.numberParts.areaCode",
+      "defaultValue": "sourceDetailsPhoneNumberFormattedNumberPartsAreaCode"
+    },
+    {
+      "name": "Source Details Phone Number Formatted Number Parts Exchange",
+      "value": "$.interaction.sourceDetails.phoneNumberFormatted.numberParts.exchange",
+      "defaultValue": "sourceDetailsPhoneNumberFormattedNumberPartsExchange"
+    },
+    {
+      "name": "Source Details Phone Number Formatted Number Parts Last Four",
+      "value": "$.interaction.sourceDetails.phoneNumberFormatted.numberParts.lastFour",
+      "defaultValue": "sourceDetailsPhoneNumberFormattedNumberPartsLastFour"
+    },
+    {
+      "name": "Source Details Phone Number Formatted Formatted International",
+      "value": "$.interaction.sourceDetails.phoneNumberFormatted.formatted.international",
+      "defaultValue": "sourceDetailsPhoneNumberFormattedFormattedInternational"
+    },
+    {
+      "name": "Source Details Phone Number Formatted Formatted National",
+      "value": "$.interaction.sourceDetails.phoneNumberFormatted.formatted.national",
+      "defaultValue": "sourceDetailsPhoneNumberFormattedFormattedNational"
+    },
+    {
+      "name": "Linked Ids Edify Person Sessions",
+      "value": "$.interaction.linkedIds.edify.personSessions",
+      "defaultValue": "linkedIdsEdifyPersonSessions"
+    },
+    {
+      "name": "Routing Details Phone Number",
+      "value": "$.interaction.routingDetails.phoneNumber",
+      "defaultValue": "routingDetailsPhoneNumber"
+    },
+    {
+      "name": "Routing Details Phone Number Formatted Type",
```

Comment:
Same idea before where removing the `Formatted`, same change with `Number Parts` and the international/national